cypress/integration/images.js

```diff
@@ -6,6 +6,12 @@ describe("images", function () {
     cy.get("#delete").click();
     cy.clearLocalForage();
     cy.wait(2000);
+
+    // remove text that Cypress seems to only sometimes think is hiding the image:
```

Comment:
The updated `/cypress/integration/images.js` might also help fix the test flakiness (#49)